packages/git_mestrelion_tools.rb

```diff
@@ -6,29 +6,48 @@
 class Git_mestrelion_tools < Package
   description 'Assorted git tools, including git-restore-mtime'
   homepage 'https://github.com/MestreLion/git-tools'
-  version '2022.12'
+  @_ver = '2022.12'
+  version "#{@_ver}-1"
   license 'GPL3'
   compatibility 'all'
   source_url 'https://github.com/MestreLion/git-tools.git'
-  git_hashtag "v#{version}"
+  git_hashtag "v#{@_ver}"
   binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: 'e008b4240a6ff1580dcb0786cc687454608d2a3dd3351da9bb7aa0a17a44cfa7',
-     armv7l: 'e008b4240a6ff1580dcb0786cc687454608d2a3dd3351da9bb7aa0a17a44cfa7',
-       i686: 'ccb3144001c111b47c4851e6b670090e8cfccee02d1a19624865dfdbdeafc47a',
-     x86_64: '9cc57f97e060f2604f8a0e1710ac118869bc27f7487017d8dd58652398fe1346'
+    aarch64: 'e9cd005b5739d50dcc7fe4e50d38c1b51e372ebc0bbf03ee36d3dbf042011db8',
+     armv7l: 'e9cd005b5739d50dcc7fe4e50d38c1b51e372ebc0bbf03ee36d3dbf042011db8',
+       i686: '79f534b94ff2c6b80aaf8eb829d3d9717f4a94c61fe3253f63bd64baa71a117f',
+     x86_64: '704dee30d25eda119b3f816f9b407f02dca6f7ee385d8ee173f2419abe7f2ba6'
   })
 
   depends_on 'git' # L
   depends_on 'python3' # R
 
+  def self.build
+    @post_merge_hook = <<~POST_MERGE_HOOK_EOF
+      #!/bin/sh
+      # This is from the git_mestrelion_tools package.
+      exec git-restore-mtime -sq 2>/dev/null
+    POST_MERGE_HOOK_EOF
+    File.write('post-merge', @post_merge_hook)
+  end
+
   def self.install
     Dir.glob('man1/*.1') do |man|
       FileUtils.install man, "#{CREW_DEST_MAN_PREFIX}/#{man}", mode: 0o644
     end
     Dir.glob('git-*') do |app|
       FileUtils.install app, "#{CREW_DEST_PREFIX}/bin/#{app}", mode: 0o755
     end
+    @crew_dest_lib_path = File.join(CREW_DEST_DIR, CREW_LIB_PATH)
+    FileUtils.install 'post-merge', File.join(@crew_dest_lib_path, '.git/hooks/post-merge'), mode: 0o755
```

Comment:
Why not add this to the main repo and commit to master?  Of course, this would require git_mestrelion_tools as a core package.

Reply:
a. `.git` is excluded from checkout and pulls.
b. It turns out that we only do `git fetch` which doesn't have a hook associated with it...

Still looking for solutions...